docs: improve quickstart & install (#395)
bytebase/bytebase.com: content/docs/en/get-started/quick-start.md

1. Start Docker.
2. Open Terminal to run the command:

#### MacOS & Linux

<pre>
curl -fsS https://raw.githubusercontent.com/bytebase/bytebase/main/quickstart/getting-started.docker-compose.yml | BB_VERSION=<version></version> docker-compose -f - up
</pre>
#### Windows
<pre>
BB_VERSION=<version></version>
</pre>
<pre>
curl -fsS https://raw.githubusercontent.com/bytebase/bytebase/main/quickstart/getting-started.docker-compose.yml | docker-compose -f - up
</pre>

<hint-block type="info">

> If the above commands doesn't work, instead of:<br/>
> https://raw.githubusercontent.com/bytebase...<br />
> use the proxy version: <br />
> https://ghproxy.com/https://raw.githubusercontent.com/bytebase...

Reviewer: ```suggestion
If the above command doesn't work, replace https://raw.githubusercontent.com/bytebase/... with the proxy version: https://ghproxy.com/https://raw.githubusercontent.com/bytebase/...
```
